yarn.lock

```diff
@@ -2997,13 +3018,16 @@
     toformat "^2.0.0"
     tslib "^2.3.1"
 
-"@uxdprotocol/uxd-client@5.1.0":
-  version "5.1.0"
-  resolved "https://npm.pkg.github.com/download/@uxdprotocol/uxd-client/5.1.0/f4583446435ef700a3f2ca4054ad078869260d0f54fa1eb0a63e5048f1b66cb1#122faf7ec36fb976f692fcaabd668177f7cee9b4"
-  integrity sha512-kePuHBnoX7BHBQDtJBk+CoCpyGxBe/tq6OhyQqeA5a3o8Tg/A2RuK0omwauglwfXgohc+EdQnnJh0JK9yjpBRA==
+"@uxd-protocol/uxd-client@1.0.0":
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@uxd-protocol/uxd-client/-/uxd-client-1.0.0.tgz#93a41fc75724bef84bcf8f03c40fadf7b963ffad"
+  integrity sha512-wI2Zg0xhRzPPDGwKgJfvJdC1fI7x96IVdXWi53u7Mw8OZSRSCS7JjoKFiJ0K5QPE4eSYKXJdQh7ChKtviK1pnQ==
   dependencies:
-    "@blockworks-foundation/mango-client" "3.3.15"
-    camelcase "^5.3.1"
+    "@blockworks-foundation/mango-client" "3.4.4"
+    "@project-serum/anchor" "0.22.1"
+    "@solana/spl-token" "0.1.8"
+    "@solana/web3.js" "1.35.1"
```

Comment:
Opportunity to align some dependencies in `package.json` assuming you use them in your code (remove them from `package.json` if not used in your code & not required in peerDependencies), no need to bother with `resolutions`.

Reply:
we really need to get work done on the dependencies... seriously